wagn/rails/controllers/card_controller.rb

```diff
@@ -135,11 +135,29 @@ def request_logger
   end
 
   def start_performance_logger
-    Card::Log::Performance.start :method=>env["REQUEST_METHOD"], :message=>env["PATH_INFO"]
+    if params[:performance_log]
+      @old_log_level = Wagn.config.log_level
+      if params[:performance_log].kind_of?(Hash) && params[:performance_log][:output] == 'console'
+        Wagn.config.log_level = :wagn
+      end
+      Card::Log::Performance.load_config params[:performance_log]
+    end
+
+    if Wagn.config.performance_logger || params[:performance_log]
+      Card::Log::Performance.start :method=>env["REQUEST_METHOD"], :message=>env["PATH_INFO"]
+    end
   end
 
   def stop_performance_logger
-    Card::Log::Performance.stop
+    if Wagn.config.performance_logger || params[:performance_log]
+      Card::Log::Performance.stop
+    end
+    if params[:perfomance_log]
+      Wagn.config.log_level = @old_log_level
+      if Wagn.config.performance_logger
+        Card::Log::Performance.load_config Wagn.config.performance_logger
+      end
+    end
```

Comment:
The performance logger can now be activated and configured for a single request with query arguments in the url. With `performance_log[output]=card` the output will be saved in the `*performance log` card otherwise it will appear in the server log.